src/main/java/unbanner/Course.java

```diff
@@ -21,7 +22,7 @@
   public List<Course> prereqs;
   public List<Course> coreqs;
   @DBRef(lazy = true)
-  public List<Student> students;
+  public List<Section> sections;
```

Comment:
I am curious what this looks like in the database, I would think that it is storing a list of sections as an embedded documents. This is probably not the behavior we would like in our app, however I am unsure on the limitations of how deeply nested references can be with Spring Data MongoDB.